ClosureKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'ClosureKit'
-  s.version = '0.0.1'
+  s.version = '0.0.2'
```

Comment:
We don't know if there's anyone relying on it. It's the responsible thing to do :)